src/javascript/app/pages/resources/economic_calendar/economic_calendar.js

```diff
@@ -0,0 +1,14 @@
+const EconomicCalendar = (() => {
+
+    const onLoad = () => {
+        $.getScript( 'https://c.mql5.com/js/widgets/calendar/widget.v3.js' )
+          .done(() => {
+              new economicCalendar({ width: '100%', height: '500px', mode: 2 }); // eslint-disable-line new-cap, no-new, no-undef
```

Comment:
Is it possible to set language as well?

Reply:
It is possible, but they don't support some of the languages that we do.